src/UBAFeeCalculator/UBAFeeSpokeCalculator.ts

```diff
@@ -4,6 +4,11 @@ import { toBN } from "../utils";
 import UBAConfig, { ThresholdBoundType } from "./UBAFeeConfig";
 import { getDepositBalancingFee, getRefundBalancingFee } from "./UBAFeeUtility";
 
+type RunningBalanceUnion = {
```

Comment:
Naming things is hard, but `Union` sticks out to me, since there's no actual union-ing happening here :D How about just `RunningBalances` or something ?

Reply:
FWIW I found that I needed the same definition for the HubPoolClient, so I've proposed to put it in interfaces as part of this PR: https://github.com/across-protocol/sdk-v2/pull/213/files#diff-d3ae5b8776de33fe7c4032263253df53ea9d0f26c6ecedf76044c7dd8b605d7c

Reply:
Resolved in the most recent SDK change.